docs/triage.md

```diff
@@ -0,0 +1,180 @@
+Triage is an important part of maintaining the health of the trivy repo.
+A well organized repo allows maintainers to prioritize feature requests, fix bugs, and respond to users facing difficulty with the tool as quickly as possible.
+
+Triage includes:
+- Labeling issues
+- Responding to issues
+- Closing issues
+
+# Daily Triage
+Daily triage has two goals:
+
+1. Responsiveness for new issues
+1. Responsiveness when explicitly requested information was provided
+
+It covers:
+
+1. Issues without a `kind/` or `triage/` label
+1. Issues without a `priority/` label
+1. `triage/needs-information` issues which the user has followed up on, and now require a response.
+
+## Categorization
+
+The most important level of categorizing the issue is defining what type it is.
+We typically want at least one of the following labels on every issue, and some issues may fall into multiple categories:
+
+- `triage/support`   - The default for most incoming issues
+- `kind/bug` - When it’s a bug or we aren’t delivering the best user experience
+
+Other possibilities: 
+- `kind/feature`- Identify new feature requests
+- `kind/testing` - Update or fix unit/integration tests
+- `kind/cleanup` - Cleaning up/refactoring the codebase
+- `kind/documentation` - Updates or additions to trivy documentation
+
+If the issue is specific to a driver for OS packages or libraries:
+
+**co/[driver for OS packages]**
+
+  - `co/alpine`
+  - `co/amazon`
+  - `co/debian`
+  - `co/oracle`
+  - `co/photon`
+  - `co/redhat`
+  - `co/suse`
+  - `co/ubuntu`
+
+**co/[driver for libraries of programming languages]** 
+
+  - `co/bundler`
+  - `co/cargo`
+  - `co/composer`
+  - `co/npm`
+  - `co/yarn`
+  - `co/pipenv`
+  - `co/poetry`
+ 
+
+**Help wanted?**
+
+`Good First Issue` - bug has a proposed solution, can be implemented w/o further discussion.
+
+`Help wanted` - if the bug could use help from a contributor
+
+
+## Prioritization
+If the issue is not `triage/support`, it needs a [priority label](https://github.com/kubernetes/community/blob/master/contributors/guide/issue-triage.md#define-priority):
+
+`priority/critical-urgent` - someones top priority ASAP, such as security issue, user-visible bug, or build breakage. Rarely used.
+
+`priority/important-soon`: in time for the next two releases (8 weeks)
```

Comment:
This commits us to a release every four weeks. I'm fine with that if you are for Trivy, although we can't with current staffing keep up a similar cadence for kube-bench / kube-hunter etc. It might be worth relaxing this text though. 

If an issue is marked as `priority/critical-urgent` or `priority/important-soon` does that mean it should also have a milestone attached? Otherwise I don't know we will have a way to track whether we're actually going to get it included within that two release timeframe.

Reply:
To be honest, I'm not sure we can keep a release every four weeks even on Trivy. Trivy is still under v1.0.0 and not maintenance phase, so I don't think we should keep a release cycle at the moment. I removed `8 weeks` 
https://github.com/aquasecurity/trivy/pull/473/commits/4c668d3417ed7b0a1ffcd6e20842f72103a0be0a

>If an issue is marked as priority/critical-urgent or priority/important-soon does that mean it should also have a milestone attached?

Good point. We should mention it. I've fixed it.
https://github.com/aquasecurity/trivy/pull/473/commits/4c668d3417ed7b0a1ffcd6e20842f72103a0be0a